core/include/vecmem/containers/impl/jagged_vector_buffer.ipp

```diff
@@ -119,4 +119,20 @@ namespace vecmem::data {
       return m_outer_host_memory.get();
    }
 
-} // namespace vecmem::data
+} // namespace data
+
+   template< typename TYPE >
+   data::jagged_vector_view< TYPE >&
+   get_data( data::jagged_vector_buffer< TYPE >& data ) {
+
+      return data;
+   }
```

Comment:
Ok, maybe I'm missing the point, but these functions exist so that you can explicitly write out the buffer -> view conversion (and use a little template deduction)?